example/lib/main.dart

```diff
@@ -108,6 +100,10 @@ class _MyHomePageState extends State<MyHomePage> {
               color: Colors.black,
               padding: EdgeInsets.all(8),
               child: Crop(
+                onChanged: (decomposition) async {
```

Comment:
Hope this `async` is alright, this `async` is required to use the `print` statement here, otherwise there will be a jank in the example during scale and rotation. 

I believe, ideally users are supposed to be doing something `async` here anyways.

What do you think?

Reply:
I think it's up to the developers to choose about async or not. For the moment i'd recommend not to use async to keep the sample project as simple as possible.

Thank you for your great efforts. I will merge your PR after this little fix.

Reply:
@xclud, pushed that fix.
Learned a lot from your reviews & suggestions. 

Thanks a lot!